src/validators/check_id.rs

```diff
@@ -36,25 +36,15 @@ pub fn validate(gtfs: &gtfs_structures::Gtfs) -> Vec<Issue> {
             related_file: None,
             geojson: None,
         });
-    let ag = if gtfs.agencies.len() <= 1 {
-        vec![]
-    } else {
-        gtfs.agencies
-            .iter()
-            .filter(|agency| !has_id(*agency))
-            .map(|agency| make_missing_id_issue(agency))
-            .collect()
-    };
-    r.chain(t)
-        .chain(c)
-        .chain(st)
-        .chain(sh)
-        .chain(ag.into_iter())
-        .collect()
+    let ag = std::iter::empty()
```

Comment:
I don't get why you chain with an empty iterator 🤔

Reply:
because I break the behaviour! If there is one Agency, we don’t need to check its ID. Thanks for spotting it ;)

Reply:
fixed